pressbooks/pressbooks-book-directory-fe
# feat: add query to the results message

Issue #433 

This PR updates the results message to include the search query in it. Previously we had a `Results: {number} books`, in the updated version we changed it slightly to be `{number} results`. The search query will be appended to the message if there is one.

This also bumps a couple of dependencies.

![image](https://github.com/pressbooks/pressbooks-book-directory-fe/assets/1761690/93263575-1ee8-456f-9c40-7a4608821eff)

![image](https://github.com/pressbooks/pressbooks-book-directory-fe/assets/1761690/6d69d6dc-664b-47fa-a966-476e9d1ae97e)

**How to test**

1. Checkout this branch
2. Run `npm install` to update dependencies
3. Run Pressbooks Directory
4. You should see `{number} results` message right above the book cards
5. Apply a search term
6. You should now see `{number} results for "{term}"` message displayed with the term you searched for
